roles/openshift_certificate_expiry/library/openshift_cert_expiry.py

```diff
@@ -613,7 +591,7 @@ def main():
         pass
 
     for kube in filter_paths(kubeconfig_paths):
-        with open(kube, 'r') as fp:
+        with io.open(kube, 'r', encoding='utf-8') as fp:
             # TODO: Maybe consider catching exceptions here?
             cfg = yaml.load(fp)
```

Comment:
I see the pattern

```python
with io.open(path, 'r', encoding='utf-8') as f:
    cfg = yaml.load(f)
    ...
```

appears here at least 3 times, maybe we could extract this into a function to load YAML configs from disk?